crates/ironrdp-pdu/src/gcc/core_data/client.rs

```diff
@@ -475,5 +475,7 @@ bitflags! {
         const SUPPORT_DYNAMIC_TIME_ZONE = 0x0200;
         const SUPPORT_HEART_BEAT_PDU = 0x0400;
         const SUPPORT_SKIP_CHANNELJOIN = 0x0800;
+        // The source may set any bits
+        const _ = !0;
```

Comment:
I think the correct fix here is to use [`from_bits_retain`](https://docs.rs/bitflags/latest/bitflags/example_generated/struct.Flags.html#method.from_bits_retain) instead of `from_bits` of `from_bits_truncate`

Reply:
I'll bump the bitflags crate. However, I feel uneasy addressing the whole #217 issue, since they are a lot of bitflags around...

Reply:
By any means, don’t address #217 as part of this PR, I was just mentioning it but it should be addressed in its own PR

Reply:
well, it fixes server connections from freerdp/mstcs clients, otherwise I wouldn't have included it here.

Reply:
Of course